P202-Training.Domain/UsersService.cs

```diff
@@ -0,0 +1,49 @@
+﻿using System.Collections.Generic;
+using AutoMapper;
+using P202.Training.Data.Repositories;
+using P202.Training.Domain.Models;
+
+namespace P202.Training.Domain
+{
+    public class UsersService : IUsersService
+    {
+        public void CreateUser(User users, IUserRepository userRepository)
+        {
+            if (users == null) return;
+            var mapUser = Mapper.Map<Data.Entities.User>(users);
+            userRepository.CreateUser(mapUser);
+        }
+
+        public void DeleteUser(int userId, IUserRepository userRepository)
+        {
+            // TODO
```

Comment:
It's ok if you wan't to leave a TODO comment but please add some more information (ie: // TODO: Implement DeleteUser method)